src/pages/PostDeliveryDetailsScreen/PostDeliveryDetailsScreen.tsx

```diff
@@ -0,0 +1,102 @@
+import { forwardRef, useContext } from 'react';
+import { Form } from 'react-final-form';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Tooltip from '@mui/material/Tooltip';
+import { Select } from 'mui-rff';
+import CustomFormSpy from 'components/CustomFormSpy';
+import FormScreens from 'constants/formScreens';
+import WizardFormContext, {
+  InitialFormValuesType,
+} from 'contexts/WizardFormContext';
+import { validateIsRequired } from 'utils/validation';
+import PostCompanies from 'constants/postCompanies';
+import POST_COMPANIES_OPTIONS from 'constants/postCompaniesOptions';
+import POST_OFFICES_OPTIONS from 'constants/postOfficesOptions';
+
+const getPostOfficeOptions = (postCompany: PostCompanies) => {
+  switch (postCompany) {
```

Comment:
Let's put all logic that defines options here. Handle a situation when there is no `postCompany` here